### PR TITLE
Utilities Section

### PR DESCRIPTION
**Pull request by tay-aitken**
- Created new utility section in the left nav
- Added in content pages for utilities section
- Added in JS pages for utilities section
- Added routes for utilities section
- Moved themes page under the style tab

### CODE REVIEW

**Comment by bsonefeld** [comment]
@tay-aitken Is this ready for review?

**Comment by tay-aitken** [comment]
Nope...I didn't change it to review. Not sure how that happened. Going to change it back. Still have to add in all the disabled work and waiting on images from Anna

**Comment by tay-aitken** [comment]
Mari there are a few things I need you to add because I couldn't figure out how to do them.

- [ ] The titles on the pages should reflect the selected section. Currently its just empty, could not figure out how to make it pull in the section name
screen shot 2018-04-14 at 10 37 45 pm

- [ ] Disabled states > all images need `brand-02` backgrounds
- [ ] Loading > small loader image needs gray outline `ui-03`
- [ ] First paragraph of each section should be bold and the component name at the beginning of the first sentence should be `brand-01`
- [ ] Truncation > 2nd image (Truncated items always... paragraph image) > shrink image so it isn't stretched out
- [ ] Truncation chart > is there anyway to make the purpose column wider so the paragraphs aren't so choppy (low priority)?
- [ ] Truncation chart > can we put the number sin `1234` inline code? I couldn't get it to work in the markdown chart

**Comment by marijohannessen** [comment]
I'll make the changes once this PR is merged in @tay-aitken , but I'll wait to merge it until next week so it doesn't get deployed this week.

**Comment by tay-aitken** [comment]
*Need to add in link from Alison under the utilities > skeleton states > code section for the codepen example*

**Comment by marijohannessen** [comment]
Everything looks good to me! 👍 Do you need @aagonzales to review it as well before merging?

File: src/content/utilities/loading/loading.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+```
+const DataComponent = ({ loading, data }) => {
+  if (loading) {
+    return <CarbonComponent.skeleton />
+  }
+  return <CarbonComponent field1={data.field1} />
+}
+```
+
+** need to add checkmark carbon icon instead of emojii :) **
```
Comment (alisonjoseph) [inline src/content/utilities/loading/loading.md]: - [ ] This table needs to get updated with correct checkmark svgs

File: src/content/utilities/loading/loading.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+/* Loading */
+<button class="bx--btn bx--btn--primary bx--skeleton"></button>
+
+/* Loaded */
+<button class="bx--btn bx--btn--primary">Button loaded</button>
+
+```
+
+If you are using our [React library](http://react.carbondesignsystem.com/) there are seperate skeleton state components available to display while data is loading.
+
```
Comment (alisonjoseph) [inline src/content/utilities/loading/loading.md]: Add link to React code sandbox demo https://codesandbox.io/s/wq264y43k8

**Comment by tay-aitken** [comment]
@aagonzales is wrapping up pictures for filtering should be done by EOD tomorrow.  Lets wait to merge this until those are added to this PR so its all in one place. Will comment on here once that is done :)

File: src/content/utilities/loading/loading.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  if (loading) {
+    return <CarbonComponent.skeleton />
+  }
+  return <CarbonComponent field1={data.field1} />
+}
+```
+
+| Component          | Vanilla | React |
+|--------------------|---------|-------|
+| Accordion          |  | ✅|
```
Comment (alisonjoseph) [inline src/content/utilities/loading/loading.md]: @marijohannessen Is there a way to add inline svgs with markdown currently in the site? Or should we just use an image tag here?

**Comment by marijohannessen** [comment]
Looks good! I'll make the changes in a separate PR! 🎉

**Pull request merged**